homeassistant/components/rfxtrx/strings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "title": "RFXCOM RFXtrx"
```

Comment:
There is no need for this, we default to the manifest title if omitted.

Reply:
Think it was added by scaffolding script?. But i'm not sure. Will drop.